src/main/java/com/faforever/client/map/MapVaultController.java

```diff
@@ -58,8 +58,7 @@
 
   private static final Logger logger = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
   private static final int TOP_ELEMENT_COUNT = 7;
-  private static final int LOAD_MORE_COUNT = 100;
-  private static final int MAX_SEARCH_RESULTS = 100;
+  private static final int LOAD_MORE_COUNT = 50;
```

Comment:
Maybe call it page count because it is not only the number of maps loaded on "more" clicked

Reply:
It isn't? It should be just that.

Reply:
yeah